src/components/SignUp/index.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --title-color: #002736;
+  --subTitle-color: #000000;
+  --background-color: #fc9e01;
+  --text-color: #eff4f6;
+  --filter-shadow-color: #b3b3b3;
+  --link-color: #0641af;
+}
+
+.signupContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.signupHeader {
+  color: #6c567b;
```

Comment:
you can add it to the color variables, I think you forgot

Reply:
revised